app/frontend/styles/components/schools_in_your_trust.scss

```diff
@@ -0,0 +1,17 @@
+.missing-information {
+  border-top: 5px solid $govuk-brand-colour;
+  padding-top: 10px;
```

Comment:
@cpjmcquillan quite rightly flagged this on my PR earlier. Using `govuk-spacing(2)` better than hardcoding these values https://design-system.service.gov.uk/styles/spacing/